static/sass/initial/_manifest.scss

```diff
@@ -4,4 +4,3 @@
 @import '~accoutrement-init/sass/init';
 @import 'root';
 @import 'type';
-@import 'site-message';
```

Comment:
I don't feel strongly. If we don't have a message we want to display right now (about our availability? or a discount? or...?), then I'd usually lean toward removing it.